File: frontend/TravelFrontEnd/src/app/Board/interface/JournalElementDTO.ts
import { TagDTO } from './ArticleData';
import { JournalDetailDTO } from './JournalDetailDTO';
export interface JournalUpDateDTO {
  title?: string,
  cover?: string,
  status: number,
  regionId?: number
  elements?: JournalElementDTO[]
  tags?: TagDTO[];
}

export interface JournalElementDTO {

  page: number,
  posX: number,
  posY: number,
  rotation: number,
  zindex: number,
  elementType: number,
  content: string,
  width: number,
  height: number,


}

File: backend/TravelWeb-API/TravelWeb-API/Models/Board/DTO/JournalElementDTO.cs
using Microsoft.AspNetCore.Mvc.RazorPages;

namespace TravelWeb_API.Models.Board.DTO
{
    public class JournalElementDTO
    {        
        public byte Page { get; set; }

        public double PosX { get; set; }

        public double PosY { get; set; }

        public double Rotation { get; set; }

        public int Zindex { get; set; }

        public byte? ElementType { get; set; }

        public string content { get; set; } = null!;

        public int Width { get; set; }
        public int Height { get; set; }

    }
}
